client_cli/src/main.rs

```diff
@@ -441,9 +441,9 @@ mod account {
                 format!("Failed to deserialize signature condition from file {}", &s);
             let content = fs::read_to_string(s).wrap_err(err_msg)?;
             let condition: Box<Expression> = json5::from_str(&content).wrap_err(deser_err_msg)?;
-            Ok(Self(SignatureCheckCondition(EvaluatesTo::new_unchecked(
-                condition,
-            ))))
+            Ok(Self(SignatureCheckCondition::new(
+                EvaluatesTo::new_unchecked(condition),
+            )))
```

Comment:
Eliding the check here isn't a good idea. Let's use the checked variant just to be safe.